src/index.html

```diff
@@ -1,13 +1,47 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+  </head>
+  <body>
+    <div data-qa="card" class="card">
+      <img 
+        src="./images/imac.jpeg" 
+        alt="iMac" 
+        class="card__image"
+      >
+      <h1 class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="card__code">Product code: 195434</p>
+      <div class="card__reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="reviews">Reviews: 5</p>
+      </div>
+      <div class="card__price">
+        <p class="card__price-text">Price</p>
+        <p class="card__price-sum">$2,199</p>
+      </div>
+      <div class="card__button">
+        <a 
+        href="/"
+        class="button"
+        data-qa="hover"
+        >
```

Comment:
Pay attention on indents
```suggestion
        <a 
          href="/"
          class="button"
          data-qa="hover"
        >
```